### PR TITLE
[docstring] Fix docstring for CanineConfig

### PR DESCRIPTION
**Pull request by Sparty**
# What does this PR do?

Fixes https://github.com/huggingface/transformers/issues/26638

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/transformers/models/canine/configuration_canine.py [hunk 1]
```diff
@@ -61,6 +61,9 @@ class CanineConfig(PretrainedConfig):
             The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
         layer_norm_eps (`float`, *optional*, defaults to 1e-12):
             The epsilon used by the layer normalization layers.
+        pad_token_id (`int`, *optional*, defaults to 0): Padding token id.
+        bos_token_id (`int`, *optional*, defaults to 57344): Beginning of stream token id.
+        eos_token_id (`int`, *optional*, defaults to 57345): End of stream token id.
```
Comment (ydshieh) [inline src/transformers/models/canine/configuration_canine.py:66]: Let's make the description, like `Padding token id.` etc., in a separate line with necessary indent.

**Comment by ydshieh** [comment]
Nice it works for you now @Sparty . One minor change and we are ready to go 🚀

File: src/transformers/models/canine/configuration_canine.py (hunk 1 above)
Comment (Sparty) [reply src/transformers/models/canine/configuration_canine.py]: Thank you for reviewing, @ydshieh. I have made the changes.

**Review by ydshieh — APPROVED** [review]
Thank you again @Sparty , let go 🚀 !

**Pull request merged**